### ISSUE TITLE
RangeError: Maximum call stack size exceeded at prepareStackTraceWithRawStackAssignment

### ISSUE THREAD

**Issue by gnestor** [atom/atom#9860]
Atom 1.4.0-dev

Steps to reproduce: Clone from master, build, and run

Result: 
```
RangeError: Maximum call stack size exceeded
    at prepareStackTraceWithRawStackAssignment (/Users/grant/Downloads/atom/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:165:50)
    at prepareStackTraceWithRawStackAssignment (/Users/grant/Downloads/atom/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:167:10)
    at prepareStackTraceWithRawStackAssignment 
...
handleSetupError @ index.js:61
```

https://github.com/atom/atom/blob/8b76c3d57cf3f74ead5fdd2eee747c3fc18f6d26/static/index.js#L61

**Comment by 50Wliu** [comment]
I happened upon this while trying to run specs through `atom --test spec` for atom/atom.  More details to come soon.

**Comment by 50Wliu** [comment]
Fixed by #9878.
